routes/index.js

```diff
@@ -1,19 +1,20 @@
 const express = require('express');
 const router = express.Router();
 const fs = require('fs');
```

Comment:
This hasn't changed, but you're no longer using this so this line can be removed.